From 20dd48fe9119fd58379b3ba15df9c2d023bb9b7f Mon Sep 17 00:00:00 2001
From: Hien Huynh <hien.huynh.px@renesas.com>
Date: Wed, 22 May 2019 13:22:24 +0700
Subject: [PATCH 70/92] drm: rcar-du: lvds: Add r8a774a1 support

The LVDS encoders on RZ/G2M SoC is similar to R-Car M3-W.
Add support for RZ/G2M (R8A774A1) SoC to the LVDS encoder driver.

Signed-off-by: Hien Huynh <hien.huynh.px@renesas.com>
---
 drivers/gpu/drm/rcar-du/rcar_lvds.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/gpu/drm/rcar-du/rcar_lvds.c b/drivers/gpu/drm/rcar-du/rcar_lvds.c
index a7894cc..c9717dd 100644
--- a/drivers/gpu/drm/rcar-du/rcar_lvds.c
+++ b/drivers/gpu/drm/rcar-du/rcar_lvds.c
@@ -1054,6 +1054,7 @@ static const struct of_device_id rcar_lvds_of_table[] = {
 	{ .compatible = "renesas,r8a77970-lvds", .data = &rcar_lvds_r8a77970_info },
 	{ .compatible = "renesas,r8a774c0-lvds",
 	.data = &rcar_lvds_r8a774c0_info },
+	{ .compatible = "renesas,r8a774a1-lvds", .data = &rcar_lvds_gen3_info },
 	{ }
 };
 
-- 
2.7.4

